stable/sematext-agent/values.yaml

```diff
@@ -0,0 +1,63 @@
+agent:
+  image:
+    repository: sematext/agent
```

Comment:
Can you please provide a commend with the source repo that this container is built from?

Reply:
Currently, agent repo is private, @otisg do we plan to make it public?

Reply:
Not at this time, @komljen